server/logger.ts

```diff
@@ -52,6 +52,22 @@ const logger = winston.createLogger({
       createSymlink: true,
       symlinkName: 'overseerr.log',
     }),
+    new winston.transports.DailyRotateFile({
+      filename: process.env.CONFIG_DIRECTORY
+        ? `${process.env.CONFIG_DIRECTORY}/logs/.machinelogs-%DATE%.json`
+        : path.join(__dirname, '../config/logs/.machinelogs-%DATE%.json'),
+      datePattern: 'YYYY-MM-DD',
+      zippedArchive: true,
```

Comment:
Is this setting useful in this transport?

Reply:
Can we just have a single day that's overwritten? Or is that not possible.

Reply:
Only having a single day is unfortunately not possible. The transport either ignores the setting or discards it when I set it to `0` or `0d` and keeps rotating the logs.
Zipping the log file up is not needed, but it's mostly for consistency with the other daily rotate transport here.